things/postgres/things.go

```diff
@@ -98,3 +98,16 @@ func (tr thingRepository) Remove(owner, id string) error {
 	tr.db.Exec(q, id, owner)
 	return nil
 }
+
+func (tr thingRepository) Identify(key string) (string, error) {
```

Comment:
Instead of naming it `Identify`, I'd rather propose that we rename all repository methods. However, that would make this PR quite big, therefore I propose that you rename only this method to: `RetrieveByKey`.

If you decide to implement the "full rename", replace `All` with `RetrieveAll()` and `One` with `RetrieveByID`.